sound/soc/sof/debug.c

```diff
@@ -812,6 +909,13 @@ int snd_sof_dbg_init(struct snd_sof_dev *sdev)
 		return err;
 #endif
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_IPC_MSG_INJECTOR)
+	err = snd_sof_debugfs_msg_inject_item(sdev, "ipc_msg_inject", 0644,
+					      &msg_inject_fops);
+	if (err < 0)
+		return err;
```

Comment:
You may want to duplicate the comment
````
/* errors are only due to memory allocation, not debugfs */
````

Some maintainers insist that we shall not stop the probe when debugfs fails, and in reviews they routinely question error handling if you don't mention what errors can happen

Reply:
Oh, thanks for the warning, I'll do that!